.github/workflows/integration-robotic-stack-tests.yml

```diff
@@ -83,7 +83,7 @@ env:
   JIRA_PASSWORD: ${{ secrets.jira_password}}
   SLACK_CHANNEL: ${{ inputs.slack_channel }}
   PROVISION_INFRA_REPO: "devops-tf-proxmox-bpg"
-  PROVISION_INFRA_VERSION: "2.0.0-3"
+  PROVISION_INFRA_VERSION: "2.0.0-10"
```

Comment:
robotics stack tests runs on marvin, the APT country could be set to pt instead of default fi